README.md

```diff
@@ -119,3 +119,63 @@ This section applies to Solidity and JavaScript projects alike.
 [solidity style guide]: ./SOLIDITY_STYLE_GUIDE.md
 [soliumrc]: ./.soliumrc.json
 [travisci]: https://travis-ci.org/
+
+## Pull Requests
+
+_The following assumes that code and other work will continue to be stored on GitHub and that ZenHub will be the management tool_.
+
+A Pull Request (PR), which is a request for changing content within a repository, can be opened at anytime.
+
+Ideally, a PR is small, limited to a single objective, and in response to a specific GitHub issue.
+
+The PR is the only place any team member is expected to consult for information about the PR or its lifecycle.
+
+### Format
+
+The PR should be connected to any issues to which it is a response. If the PR resolves an issue, that should be indicated in the body of the PR (e.g., "Resolves #70", "Fixes #71", etc.).
+
+The PR's title and description should collectively address the following:
+- what is the purpose of these changes
```

Comment:
This document uses asterisks, not dashes, to specify unordered lists.

Reply:
@schemar: Corrected!